demo/29-numbered-lists.ts

```diff
@@ -1,54 +1,80 @@
 // Numbered lists
 // Import from 'docx' rather than '../build' if you install from npm
 import * as fs from "fs";
-import { AlignmentType, Document, Packer, Paragraph } from "../build";
+import { AlignmentType, File, Numbering, Packer, Paragraph } from "../build";
 
-const doc = new Document({
-    numbering: {
-        config: [
-            {
-                levels: [
-                    {
-                        level: 0,
-                        format: "upperRoman",
-                        text: "%1",
-                        alignment: AlignmentType.START,
-                        style: {
-                            paragraph: {
-                                indent: { left: 720, hanging: 260 },
-                            },
-                        },
-                    },
-                ],
-                reference: "my-crazy-reference",
-            },
-            {
-                levels: [
-                    {
-                        level: 0,
-                        format: "decimal",
-                        text: "%1",
-                        alignment: AlignmentType.START,
-                        style: {
-                            paragraph: {
-                                indent: { left: 720, hanging: 260 },
-                            },
-                        },
-                    },
-                ],
-                reference: "my-number-numbering-reference",
-            },
-        ],
+const doc = new File({});
```

Comment:
The previous example used a `Document`, you're using a `File`, wouldn't it be better to stick to the old format or is that impossible somehow?

Reply:
Not sure what is the difference between `File` and `Document` but `Numbering` is only available through `File` that's why I changed it

Reply:
I see, good point